build.gradle

```diff
@@ -88,7 +88,7 @@ configure(subprojects) {
         java {
             target 'src/**/*.java'
             targetExclude '**/test/resources/**'
-            googleJavaFormat('1.22.0').reorderImports(false).skipJavadocFormatting()
+            googleJavaFormat('1.22.0').skipJavadocFormatting()
```

Comment:
Seems like the default is false

https://github.com/diffplug/spotless/blob/13d24aea2bad9059c8c214ee8fa0d2fbc3e2c628/lib/src/main/java/com/diffplug/spotless/java/GoogleJavaFormatStep.java#L33